src/pudl/etl.py

```diff
@@ -482,6 +503,7 @@ def etl(  # noqa: C901
         sqlite_dfs.update(_etl_ferc1(datasets["ferc1"], pudl_settings))
     if datasets.get("eia", False):
         sqlite_dfs.update(_etl_eia(datasets["eia"], ds_kwargs))
+        sqlite_dfs.update(_etl_eia_bulk_elec(datasets["eia_bulk_elec"], ds_kwargs))
```

Comment:
@zaneselvans I tried to imitate the other function calls, but this gives a KeyError.  I don't know the right way to access the datastore. Is this something you know off the top of your head or should I dive in?

Reply:
What's it giving they KeyError on, the reference to `datasets["eia_bulk_elec"]`? That `datasets` dictionary contains the validated ETL settings (which are based on the settings YAML file used to run the ETL). There would only be an `eia_bulk_elec` entry there if you had created a new `dataset` section in the settings file, and defined a new ETL Settings class for the settings associated with this data source.

Are there settings associated with this dataset? If there's just one way to load it, and it should always be loaded whenever the EIA data is being loaded (which seems like it's probably right) then either `etl_eia_bulk_elec()` should be called from within the `_etl_eia()` function, or it could be called separately here inside this if statement.

The proximate problem here though is probably that you're calling `_etl_eia_bulk_elec()` with two positional arguments, and the function only expects one argument, with that argument being the `ds_kwargs`.

Reply:
oops, I didn't even notice the positional args problem because the KeyError hit first. There are no settings for this yet, so I'll just remove the reference to `datasets`